proto/data.proto

```diff
@@ -324,3 +324,26 @@ message TimeSeriesData {
   // Datapoints representing one or more measurements taken from the variable.
   repeated TimeSeriesDatapoint datapoints = 3;
 }
+
+// MVCCStats tracks byte and instance counts for:
+//  - Live key/values (i.e. what a scan at current time will reveal;
+//    note that this includes intent keys and values, but not keys and
+//    values with most recent value deleted)
+//  - Key bytes (includes all keys, even those with most recent value deleted)
+//  - Value bytes (includes all versions)
+//  - Key count (count of all keys, including keys with deleted tombstones)
+//  - Value count (all versions, including deleted tombstones)
+//  - Intents (provisional values written during txns)
+message MVCCStats {
```

Comment:
I found this useful as well:
https://en.wikipedia.org/wiki/Multiversion_concurrency_control